<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraWindow</class>
 <widget class="QMainWindow" name="CameraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>jkam</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="IntegrateROI" name="history_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="CameraControlWidget" name="camera_control_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>203</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>203</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QStackedWidget" name="view_stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>565</width>
        <height>422</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="imageview_page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="ImageViewEditor" name="imageview_widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="absorption_view_page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="AbsorptionViewWidget" name="absorption_view_widget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IntegrateROI</class>
   <extends>QWidget</extends>
   <header>AnalysisWidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AbsorptionViewWidget</class>
   <extends>QWidget</extends>
   <header>absorption_view_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageViewEditor</class>
   <extends>QWidget</extends>
   <header>imagevieweditor</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CameraControlWidget</class>
   <extends>QWidget</extends>
   <header>cameracontrolwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="cam_mode_buttonGroup"/>
 </buttongroups>
</ui>
